swarm.go

```diff
@@ -75,6 +77,7 @@ type Swarm struct {
 	config            *api.Config        // swarm configuration
 	api               *api.API           // high level api layer (fs/manifest)
 	dns               api.Resolver       // DNS registrar
```

Comment:
What is the difference between ens and dns?

Reply:
dns is the wrapper, for resolving multiple tld resolvers, it applies for ens but could be used for other resolvers.
on the contrary rns only resolver rsk domains with one tld.
this could be addressed in this issue #1991